apps/nextjs-website/react-components/components/Editorial/Content.tsx

```diff
@@ -1,27 +1,54 @@
 import React from 'react';
-import { useTheme } from '@mui/material';
+import { useTheme, Theme } from '@mui/material';
 import Stack from '@mui/material/Stack';
 import { EditorialContentProps } from '../../types/Editorial/Editorial.types';
 import { Body, Title } from '../common/Common';
 import { Eyelet } from './Editorial.helpers';
 import { TextColor, ExtraTextColor } from '../common/Common.helpers';
 
+const customStyles = (theme: Theme) => {
+  const { palette } = theme;
+  return {
+    dark: {
+      'a': {
+        color: `${palette.primary.contrastText} !important`,
+        fontWeight: '700 !important',
+        textDecorationColor: `${palette.primary.contrastText} !important`,
+        '&:hover': {
+          color: `${palette.primary.contrastText} !important`,
+        },
+      },
+    },
+    light: {
+      'a': {
+        color: `${palette.primary.main} !important`,
+        fontWeight: '700 !important',
+        textDecorationColor: `${palette.primary.main} !important`,
+        '&:hover': {
+          color: `${palette.primary.main} !important`,
+        },
+      },
+    },
+  };
+};
```

Comment:
```suggestion
const customStyles = ({ palette }: Theme, theme: 'light' | 'dark') =>
  theme === 'dark'
    ? {
        a: {
          color: `${palette.primary.contrastText} !important`,
          fontWeight: '700 !important',
          textDecorationColor: `${palette.primary.contrastText} !important`,
          '&:hover': {
            color: `${palette.primary.contrastText} !important`,
          },
        },
      }
    : {
        a: {
          color: `${palette.primary.main} !important`,
          fontWeight: '700 !important',
          textDecorationColor: `${palette.primary.main} !important`,
          '&:hover': {
            color: `${palette.primary.main} !important`,
          },
        },
      };
```
Integrate theme into customStyles and return only needed styles.